components/FormComponents/GitHubUserNameInput.tsx

```diff
@@ -8,12 +8,13 @@ const GitHubUserNameInput: React.FC<FormState> = ({
   setFormData,
 }) => {
   return (
-    <div className="flex flex-col mt-10 max-w-2xl">
-      <label htmlFor="name" className="mb-2 md:text-lg lg:text-xl">
+    <div className="flex flex-col mt-10 max-w-2xl" role="Github Username">
```

Comment:
There are two opening div tags but only one closing div tag at the end. Please fix it.

Reply:
Sure